cmd/kubeseal/main.go

```diff
@@ -2,16 +2,18 @@ package main
 
 import (
 	"crypto/rsa"
+	"encoding/json"
 	"errors"
 	goflag "flag"
 	"fmt"
 	"io"
 	"io/ioutil"
-	"k8s.io/apimachinery/pkg/util/net"
 	"net/http"
 	"os"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/util/net"
```

Comment:
Remove the whitespace after this import (should be combined with the rest of the 3rd-party imports)